### ISSUE TITLE
Markdown format is bugged out and looks bad

### ISSUE THREAD

**Issue by eliasriise** [td-org-uit-no/tdctl-frontend#190]
![image](https://github.com/td-org-uit-no/tdctl-frontend/assets/43967847/8b9a6d36-95bf-444d-aa3b-c96d4ee02c1f)
![image](https://github.com/td-org-uit-no/tdctl-frontend/assets/43967847/ef6f3220-c33f-44db-a1c2-1445ce61bd24)
![image](https://github.com/td-org-uit-no/tdctl-frontend/assets/43967847/10324411-643c-468b-9d1c-30856ac7e4d3)
1. First image shows the raw markdown text
2. Second image shows how it should look like
3. Last image shows how it _actually_ looks like

To sum up, there is an enormous amount of newline space after headers, and the header text is maybe overall a bit oversized as well?

**Comment by otytlandsvik** [comment]
One possible solution is to create a wrapper component and [specify style](https://github.com/remarkjs/react-markdown?tab=readme-ov-file#appendix-b-components) of the elements created by ReactMarkdown.
